platform/sun50i/devices.c

```diff
@@ -36,6 +39,16 @@ static struct device msgbox = {
 	.irqdev   = &r_intc,
 };
 
+static struct device pio = {
+	.name     = "pio",
+	.clock    = CCU_GATE(CCU_GATE_PIO) | CCU_RESET(CCU_RESET_PIO),
+	.clockdev = &ccu,
+	.drv      = &sunxi_pio_driver,
+	.drvdata  = BITMASK(1, 7), /*< Physically implemented PIO ports (1-7).
```

Comment:
Because you're using this macro (and `BIT` below), you should explicitly include `util.h`.

Since this doesn't fit in the column limit, maybe remove "PIO" (since this evident from context) or move the comment above the `.drvdata` line (and in that case remove the angle bracket).

Reply:
Done (PIO removed).